src/runtime_config/default_actions.js

```diff
@@ -418,4 +418,29 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "add_node_code",
```

Comment:
Ditto, should we rename the existing hardcoded C++ proposals to match these too?

Reply:
Let's keep the Lua gov as compatible as possible, as discussed.